presto-spi/src/main/java/io/prestosql/spi/connector/EmptySplitPageSource.java

```diff
@@ -36,7 +35,7 @@ public void deleteRows(Block rowIds)
     @Override
     public CompletableFuture<Collection<Slice>> finish()
     {
-        return completedFuture(ImmutableList.of());
+        return completedFuture(Collections.emptyList());
```

Comment:
undo. Guava is smart when doing `copyOf` if underlying list intance is also immutable list

Reply:
I couldn't use Guava here due to the move to presto-spi
Would something else be more appropriate ?

Reply:
Ah, your are right.